### PR TITLE
updates brain damage lines

### PR DESCRIPTION
**Pull request by Tacolizard**
https://github.com/tgstation/tgstation/pull/28288#issuecomment-307503748

### CODE REVIEW

**Comment by optimumtact** [comment]
Fix your title and pr message to be descriptive, a maintainer should not have to click through a link to understand what you're fixing please.

**Comment by optimumtact** [comment]
Thank you

**Comment by ExcessiveUseOfCobblestone** [comment]
this will not be enjoyable to hear memes spouted on radio uncontrollably

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -121,6 +124,17 @@
...
-    ]
+    ],
+	
+	"bug": [
+		"",
+		"IS TIS A BUG??",
+		"SI IST A BUGG/"
+	],
+	
+	"semicolon": [
```
Comment (octareenroon91) [inline strings/brain_damage_lines.json]: add `":h" to this list.

File: strings/brain_damage_lines.json [hunk 2]
```diff
@@ -31,34 +31,37 @@
...
+        "@pick(semicolon)I'VE GOT BALLS OF STEEL",
         "NO I'M ONNA KILL YOU MOTHERFUCKER OLD STYLE",
         "i will snatch erry motherfucker birthday",
-        "the ai and borgs are mettacomming I think",
+        "@pick(semicolon)the ai and borgs are mettacomming I think",
         "u just did the world a little bit more sad place for someone",
         "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?",
-        "N-NYAAAAAA~"
+        "@pick(semicolon)N-NYAAAAAA~",
+		"@pick(bug)"
```
Comment (RemieRichards) [inline strings/brain_damage_lines.json]: ,

**Comment by optimumtact** [comment]
you must use spaces and not tabs in this file, it is a json file, formatted with prettyprint

**Comment by ExcessiveUseOfCobblestone** [comment]
Does this not break the freeze or is he using his organ pr ticket on this lol

**Comment by Tacolizard** [comment]
@ExcessiveUseOfCobblestone people with brain damage don't say their lines super often, and some brain damage lines already had semicolons

File: strings/brain_damage_lines.json [hunk 3]
```diff
@@ -31,34 +31,37 @@
...
+        "@pick(semicolon)I'VE GOT BALLS OF STEEL",
         "NO I'M ONNA KILL YOU MOTHERFUCKER OLD STYLE",
         "i will snatch erry motherfucker birthday",
-        "the ai and borgs are mettacomming I think",
+        "@pick(semicolon)the ai and borgs are mettacomming I think",
         "u just did the world a little bit more sad place for someone",
         "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?",
-        "N-NYAAAAAA~"
+        "@pick(semicolon)N-NYAAAAAA~",
+		"@pick(bug)",
```
Comment (Cyberboss) [inline strings/brain_damage_lines.json]: Missed

File: strings/brain_damage_lines.json [hunk 4]
```diff
@@ -93,7 +96,8 @@
     "random_gibberish": [
         "g",
         "squid",
-        "r"
+        "r",
+		"carbon dioxide"
```
Comment (Cyberboss) [inline strings/brain_damage_lines.json]: again

File: strings/brain_damage_lines.json [hunk 5]
```diff
@@ -121,6 +125,19 @@
...
         "sheadow lings",
         "abdoocters"
+    ],
+	
+    "bug": [
+        "",
+        "IS TIS A BUG??",
+        "SI IST A BUGG/",
+		"IS TIHS AN BUUG"
+    ],
```
Comment (Cyberboss) [inline strings/brain_damage_lines.json]: again

File: strings/brain_damage_lines.json [hunk 6]
```diff
@@ -121,6 +125,19 @@
...
+	
+    "semicolon": [
+        "",
+        ";",
+		".h"
```
Comment (Cyberboss) [inline strings/brain_damage_lines.json]: again

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Just run this through the prettifier

**Comment by Tacolizard** [comment]
@Cyberboss i would but I'm away from home and my computer currently

File: strings/brain_damage_lines.json [hunk 7]
```diff
@@ -121,6 +125,19 @@
         "treaitors",
         "sheadow lings",
         "abdoocters"
+    ],
+	
+    "bug": [
+        "",
+        "IS TIS A BUG??",
+        "SI IST A BUGG/",
+		"BUG!!!"
```
Comment (Cyberboss) [inline strings/brain_damage_lines.json]: Missed one

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by XDTM** [comment]
Now i want to hide a lot of monkeymen with brain damage and a radio somewhere inside a wall

**Comment by Tacolizard** [comment]
@XDTM you could do the same thing using handheld radios

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**